### ISSUE TITLE
Site behavior with "NoScript" browser add-ons/plugins

### ISSUE THREAD

**Issue by ehenneken** [adsabs/bumblebee#1252]
Bumblebee, as expected, does not load when a plugin like NoScript for Firefox is active. Whitelisting by selecting "Allow all this page" (in NoScript) still leaves Bumblebee non-functional because `cloudflare.com` and `mathjax.org` still get blocked. Is it possible to serve the JavaScript libraries, currently retrieved from those domains, from our own domain?

**Comment by thostetler** [comment]
Yeah there are several being served from CDNs, we could bundle these libs up during the build instead.
